main/forms.py

```diff
@@ -59,26 +59,24 @@ class CostRecoveryForm(forms.Form):
     )
 
 
-class ProjectForm(forms.ModelForm):  # type: ignore [type-arg]
-    """Form to create and edit Project instances."""
+class FundingForm(forms.ModelForm):  # type: ignore [type-arg]
```

Comment:
Why are you modifying the `ProjectForm` instead of creating a separate `FundingForm`? You should not be changing the existing functionality.